src/audio/sequence.c

```diff
@@ -43,7 +43,11 @@ void Audio_StartSequence(u8 seqPlayerIndex, u8 seqId, u8 seqArgs, u16 fadeInDura
 
     if (!gStartSeqDisabled || (seqPlayerIndex == SEQ_PLAYER_SFX)) {
         seqArgs &= 0x7F;
+#ifdef OOT_DEBUG
         if (seqArgs == 0x7F) {
+#else
+        if (0) {
```

Comment:
personally I really don't like it when only one brace is ifdef'd, I think it's unreadable. One alternative is something like
```
#ifdef OOT_DEBUG
#define FOO(seqArgs) (seqArgs == 0x7F)
#else
#define FOO(seqArgs) false
#endif
```
but maybe I should give up on this crusade and it's more trouble than it's worth. What do you think?

Reply:
I think this would be cleaned up by https://github.com/zeldaret/oot/pull/1658 so never mind